docs/100.yaml

```
@@ -327,6 +327,10 @@
   title  : Desargues’s Theorem
 88:
   title  : Derangements Formula
+    decls  :
+    - num_derangements_recursive
+    - num_derangements_sum
+  author : Henry Swanson
```

Check the linter: something is broken here.

---

```suggestion
88:
  title  : Derangements Formula
  decls  :
    - num_derangements_recursive
    - num_derangements_sum
  author : Henry Swanson
```

i think YAML cares about indents